docker/Dockerfile-cache.binary

```
@@ -0,0 +1,30 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
+WORKDIR /app
+
+FROM chef AS planner
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef AS builder 
+COPY --from=planner /app/recipe.json recipe.json
+# Build dependencies - this is the caching Docker layer!
+RUN cargo chef cook --release --recipe-path recipe.json
+# Build application
+COPY . .
+RUN cargo build --release
```

I think we still need to include debug symbols. Otherwise we'll probably not get a useful stack trace on panics.

```suggestion
RUN CARGO_PROFILE_RELEASE_DEBUG=1 cargo build --release
```